src/main/java/com/faforever/client/fx/DualStringListCell.java

```diff
@@ -48,12 +26,15 @@ protected void updateItem(T item, boolean empty) {
         setText(null);
         setGraphic(null);
       } else {
-        left.setText(functionLeft.apply(item));
-        right.setText(functionRight.apply(item));
+        if (dualStringListCellController == null) {
+          dualStringListCellController = uiService.loadFxml("theme/dual_string_list_cell.fxml");
```

Comment:
This is not very performance friendly.
The JavaFX documentation (in some parent class) explicitely states that updateItem should ONLY reuse ui elements not create or recreate them.

Reply:
yes, this is essentially the same as u did...